src/plugins/data_views/common/content_management/v1/cm_services.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { schema } from '@kbn/config-schema';
+import type { ContentManagementServicesDefinition as ServicesDefinition } from '@kbn/object-versioning';
+import {
+  savedObjectSchema,
+  objectTypeToGetResultSchema,
+  createOptionsSchemas,
+  updateOptionsSchema,
+  createResultSchema,
+  searchOptionsSchemas,
+} from '@kbn/content-management-utils';
+import { serializedFieldFormatSchema, fieldSpecSchema } from '../../schemas';
+
+const dataViewAttributesSchema = schema.object(
+  {
+    title: schema.string(),
+    type: schema.maybe(schema.string()),
+    timeFieldName: schema.maybe(schema.string()),
+    sourceFilters: schema.maybe(
+      schema.arrayOf(
+        schema.object({
+          value: schema.string(),
+        })
+      )
+    ),
+    fields: schema.maybe(schema.arrayOf(fieldSpecSchema)),
+    typeMeta: schema.maybe(schema.object({}, { unknowns: 'allow' })),
+    fieldFormatMap: schema.maybe(schema.recordOf(schema.string(), serializedFieldFormatSchema)),
+    fieldAttrs: schema.maybe(
+      schema.recordOf(
+        schema.string(),
+        schema.object({
+          customLabel: schema.maybe(schema.string()),
+          count: schema.maybe(schema.number()),
+        })
+      )
+    ),
+    allowNoIndex: schema.maybe(schema.boolean()),
+    runtimeFieldMap: schema.maybe(schema.any()),
+    name: schema.maybe(schema.string()),
+  },
+  { unknowns: 'forbid' }
+);
+
+const dataViewSavedObjectSchema = savedObjectSchema(dataViewAttributesSchema);
+
+const dataViewCreateOptionsSchema = schema.object({
+  id: createOptionsSchemas.id,
+  initialNamespaces: createOptionsSchemas.initialNamespaces,
+});
+
+const dataViewSearchOptionsSchema = schema.object({
+  searchFields: searchOptionsSchemas.searchFields,
+  fields: searchOptionsSchemas.fields,
+});
+
+const dataViewUpdateOptionsSchema = schema.object({
+  version: updateOptionsSchema.version,
+  refresh: updateOptionsSchema.refresh,
+  retryOnConflict: updateOptionsSchema.retryOnConflict,
+});
+
+// Content management service definition.
+// We need it for BWC support between different versions of the content
+export const serviceDefinition: ServicesDefinition = {
+  get: {
+    out: {
```

Comment:
There's no reason since the `in` portion is just the id of the doc we're getting. That said, if we needed it we could add it later, although I admit it would be nice to have more clarity and agreement around what it would take to introduce new transforms.